ios/app/JitsiMeetBroadcast Extension/SampleHandler.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  SampleHandler.swift
+//  Broadcast Extension
+//
+//  Created by Alex-Dan Bumbu on 22/03/2021.
+//  Copyright © 2021 Atlassian Inc. All rights reserved.
+//
+
+import ReplayKit
+
+private enum Constants {
+    static let appGroupIdentifier = "group.org.jitsi.meet.appgroup"
+}
+
+class SampleHandler: RPBroadcastSampleHandler {
+    
+    private var clientConnection: SocketConnection?
+    private var uploader: SampleUploader?
+    
+    private var frameCount: Int = 0
+    
+    var socketFilePath: String {
+      let sharedContainer = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: Constants.appGroupIdentifier)
+        
+        return sharedContainer?.appendingPathComponent("rtc_SSFD").path ?? ""
+    }
+    
+    override init() {
+      super.init()
+        if let connection = SocketConnection(filePath: socketFilePath) {
+          clientConnection = connection
+          setupConnection()
+          
+          uploader = SampleUploader(connection: connection)
+        }
+    }
+
+    override func broadcastStarted(withSetupInfo setupInfo: [String: NSObject]?) {
+        // User has requested to start the broadcast. Setup info from the UI extension can be supplied but optional.
+        print("broadcast started")
```

Comment:
Can we use our logger here?

Reply:
cocoa lumberjack instances are not shared between the app and the extension. If we want this we need to implement a custom mechanism for forwarding messages from the extension to the app.